eHealthAfrica/ng-admin
# Men 372

Summary of this PR:

- Extends the pagination directive to support displaying page buttons even when only a single page is available.


How to test this PR:

- Use the blog example or LMIS-Admin
- Run the app normally and observe the pagination's behaviour
- Go to the `entity.ListView()` and set the property `alwaysDisplayPagination` to `true`
 i.e: `entity.ListView().alwaysDisplayPagination = true`
-Rebuild the app and run it then observe the pagination's behaviour
